pymeasure/instruments/agilent/agilent33220A.py

```diff
@@ -258,8 +266,8 @@ def wait_for_trigger(self, timeout=3600, should_stop=lambda: False):
 
             if timeout != 0 and time() - t0 > timeout:
                 raise TimeoutError(
-                    "Timeout expired while waiting for the Agilent 33220A" +
-                    " to finish the triggering."
+                    "Timeout expired while waiting for the Agilent 33220A"
+                    + " to finish the triggering."
```

Comment:
Actually, the `+`  is not needed as several strings in a parenthesis, without a comma in between, are considered a single string.